### PR TITLE
Add layers for Delay Tolerant Networking Protocols

### PR DESCRIPTION
**Pull request by keithlscott**
Adds layers (in scapy/contrib) for Licklider Transmission Protocol (LTP, RFC5326) and Bundle Protocol (RFC5050).

### CODE REVIEW

**Comment by gpotter2** [comment]
@keithlscott Hi, this looks great !
Few comments:
- You copied config.py
- Could you add .uts tests (have a look at other .uts, the format is easy and you may copy the 'do_tests' functions in it) ?

Thanks !

**Comment by keithlscott** [comment]
OK, I'll see about some .uts tests and -- resubmit the pull request?
What's the right thing to do?

  v/r,

   --keith

**Comment by gpotter2** [comment]
You may just append a commit to this PR :)

**Comment by gpotter2** [comment]
@keithlscott Hi ! I'm currently adding some unit tests for this PR. I have updated it quite a bit ^^
Would you be able to share a PCAP ? It would greatly help us

thanks !

**Comment by keithlscott** [comment]
Hey, thanks.  I apologize for being non-responsive, I've been slammed...
I will endeavor to provide some pcaps by Monday.

      v/r,

    --keith

**Comment by guedou** [comment]
@keithlscott any update on this PR?

File: config.py [hunk 1]
```diff
@@ -0,0 +1,483 @@
+## This file is part of Scapy
```
Comment (guedou) [inline config.py]: Please remove this file.

File: scapy/config.py [hunk 2]
```diff
@@ -449,7 +449,8 @@ class Conf(ConfClass):
                    "mobileip", "netbios", "netflow", "ntp", "ppp", "pptp",
                    "radius", "rip", "rtp", "skinny", "smb", "snmp",
                    "tftp", "x509", "bluetooth", "dhcp6", "llmnr",
-                   "sctp", "vrrp", "ipsec", "lltd", "vxlan", "eap"]
+                   "sctp", "vrrp", "ipsec", "lltd", "vxlan", "eap",
+                   "ltp", "bp"]
```
Comment (guedou) [inline scapy/config.py]: This not necessary since you moved the layers to `contrib/`.

**Comment by guedou** [comment]
PR fixed by #1016

**Pull request closed without merging**